src/test/java/org/opensearch/security/sanity/tests/InvalidAdminPasswordIT.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.sanity.tests;
+
+import org.hamcrest.MatcherAssert;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import org.opensearch.client.Request;
+import org.opensearch.client.Response;
+import org.opensearch.client.ResponseException;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+public class InvalidAdminPasswordIT extends SecurityRestTestCase {
+
+    @BeforeClass
+    public static void setUpInvalidAdminCredentials() {
+        useAdminAsPassword = true;
+    }
+
+    @AfterClass
+    public static void restoreValidAdminCredentials() {
+        useAdminAsPassword = false;
+    }
```

Comment:
This is hard to follow, instead could we encode the username/password as a basic auth header in the way that gets a 401 and then the other where it works as expected?

Reply:
since the client is build before the class is invoked, the only way to build client with invalid credentials is using @BeforeClass and then setting invalid admin password.

Reply:
This seems to be due to the construction of the rest client, could you create a follow up issue? Not being able to change credentials during a test seems like it will hamper many commonly tested scenarios in our codebase.